Smart Village App/views/stream/widgets/wallStreamFilterNavigation.php

```diff
@@ -0,0 +1,48 @@
+<?php
+/**
+ * @link https://www.humhub.org/
+ * @copyright Copyright (c) 2018 HumHub GmbH & Co. KG
+ * @license https://www.humhub.com/licences
+ *
+ */
+
+use humhub\modules\stream\widgets\WallStreamFilterNavigation;
+use humhub\modules\ui\filter\widgets\FilterPanel;
+use humhub\widgets\Button;
+use yii\helpers\Html;
+
+/* @var $this \humhub\modules\ui\view\components\View */
+/* @var $panels [] */
+/* @var $options [] */
+
+$leftPanelBlocks = isset($panels[WallStreamFilterNavigation::PANEL_POSITION_LEFT]) ? $panels[WallStreamFilterNavigation::PANEL_POSITION_LEFT] : null;
+$centerPanelBlocks = isset($panels[WallStreamFilterNavigation::PANEL_POSITION_CENTER]) ? $panels[WallStreamFilterNavigation::PANEL_POSITION_CENTER] : null;
+$rightPanelBlocks = isset($panels[WallStreamFilterNavigation::PANEL_POSITION_RIGHT]) ? $panels[WallStreamFilterNavigation::PANEL_POSITION_RIGHT] : null;
+
+?>
+
+<?= Html::beginTag('div', $options) ?>
+
+<div class="wall-stream-filter-root nav-tabs">
+    <div class="wall-stream-filter-head clearfix">
+        <div class="wall-stream-filter-bar"></div>
+        <?= Button::asLink(Yii::t('ContentModule.base', 'Filter') . '<b class="caret"></b>')
+            ->cssClass('wall-stream-filter-toggle')->icon('fa-filter')->sm()->style('pa') ?>
+    </div>
+    <div class="wall-stream-filter-body" style="display:none">
+        <div class="filter-root">
+            <div class="row">
+                <?= FilterPanel::widget(['blocks' => $leftPanelBlocks, 'span' => count($panels)])?>
+                <?= FilterPanel::widget(['blocks' => $centerPanelBlocks, 'span' => count($panels)])?>
+                <?= FilterPanel::widget(['blocks' => $rightPanelBlocks, 'span' => count($panels)])?>
+            </div>
+        </div>
+    </div>
+</div>
+
+<?= Html::endTag('div') ?>
+
+<script>
+    let filterRemove = document.getElementById("wall-stream-filter-nav");
```

Comment:
where is the `wall-stream-filter-nav` element located? it is not here in this file.

Reply:
Hi @donni106 
The 'wall-stream-filter-nav' is the id of the  top most 'div' tag of filer section.

![image](https://user-images.githubusercontent.com/103625795/178899559-6de1d700-c285-4a37-b2ea-215bc0e31f87.png)

And in the code it is wrap-up in $option variable.

![image](https://user-images.githubusercontent.com/103625795/178899632-d035db04-88bc-422a-a613-bf90b98eaf49.png)

Reply:
yes